lib/registries/addon/components/registration-form-view/x-options/styles.scss

```
@@ -0,0 +1,23 @@
+.OptionsList {
+    list-style: none;
+    margin-left: 0;
+    padding-left: 40px;
```

I apologize, this should have been in the ticket description, but part of the impetus for this change was to align the bullet/check with the text above and below it.
```suggestion
    padding-left: 20px;
```